monitoring/README.md

```diff
@@ -86,14 +86,20 @@ are present in `USER_DIR`:
 * `monitoring/user-values-prom-operator.yaml`
 * `monitoring/user-values-pushgateway.yaml`
 
+You can modify two types of values to customize your deployment:
+- Environment variables: These variables contain uppercase characters and underscores (for example, `LOG_COLOR_ENABLE=`), and can be specified in the `user.env` file or on the command line. Specifying the variables in `user.env` is recommended, in order to maintain a consistent set of values for future deployments.
+- Helm chart parameters: These parameters are organized into a hierarchical structure and contain mixed case characters (for example, `persistentVolume:storageClass`). These parameters are specified in `.yaml` configuration files.
```

Comment:
See comment above about Environment Variables.

Reply:
Removed information about case and characters in variables